pkg/sql/sem/builtins/generator_builtins.go

```diff
@@ -465,6 +466,18 @@ var generators = map[string]builtinDefinition{
 			volatility.Volatile,
 		),
 	),
+	"crdb_internal.ranges_in_span": makeBuiltin(genProps(),
+		makeGeneratorOverload(
+			tree.ParamTypes{
+				{Name: "start_key", Typ: types.Bytes},
+				{Name: "end_key", Typ: types.Bytes},
```

Comment:
Minor feature request: If you add another overload that takes a single parameter that is a byte array and then do something like:

```
				arr := tree.MustBeDArray(args[0])
				if arr.Len() != 2 {
					return nil, errors.New("expected an array of two elements")
				}
				startKey := []byte(tree.MustBeDBytes(arr.Array[0]))
				endKey := []byte(tree.MustBeDBytes(arr.Array[1]))
```

Then this function composes well with `crdb_internal.tenant_span`, crdb_internal.table_span`, and `crdb_internal.index_span`.

Reply:
Added byte array overload